src/axom/multimat/examples/calculate.cpp

```diff
@@ -4519,6 +4519,13 @@ int main(int argc, char** argv)
   //using RelationSetType = slam::RelationSet<RelType, SetType, SetType>;
   using RelationSetType = MultiMat::RelationSetType;
 
+  // Note: we keep the below mappings within the lifetime context of main()
+  // This avoids an issue with static destruction order wrt Umpire.
+  std::unordered_map<const ProductSetType*, ConcreteProdSet> concretizedProdSets;
+  std::unordered_map<const RelationSetType*, ConcreteRelationSet> concretizedRelSets;
+  g_concretizedProdSets = &concretizedProdSets;
+  g_concretizedRelSets = &concretizedRelSets;
```

Comment:
Is this something `multimat` users will typically have to worry about? Or just an implementation detail of how this example was implemented?

Reply:
This should not be needed by end users -- this just supports our testing of various Slam template specializations for benchmarking Multimat access performance.